tests/dummy/app/models/fake-model.js

```diff
@@ -133,6 +137,26 @@ export default DS.Model.extend(Validator,{
     },
     otherFake:{
       relations: ['belongsTo']
+    },
+    otherCustomValidation: {
+      custom: {
+        validation: function(key, value){
+          return value.toString().length === 5 ? true : false;
+        },
+        message: function(){
```

Comment:
@esbanarango  I am going to pass the key,value and model, because it could be helpful to have access to some model properties to generate the message.